modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/LogicalRelImplementor.java

```diff
@@ -463,7 +463,11 @@ public LogicalRelImplementor(
     @Override public Node<Row> visit(IgniteSort rel) {
         RelCollation collation = rel.getCollation();
 
-        SortNode<Row> node = new SortNode<>(ctx, rel.getRowType(), expressionFactory.comparator(collation));
+        Supplier<Integer> limit = (rel.fetch == null) ? null : expressionFactory.execute(rel.fetch);
```

Comment:
Let's use the same variable name for `fetch`

Reply:
Fixed